src/components/atoms/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { DropdownButton } from "react-bootstrap";
+import classNames from "classnames";
+
+import { ContainerClassName } from "types/utility";
+
+import "./Dropdown.scss";
+
+export interface DropdownProps extends ContainerClassName {
+  id: string;
+  title: string;
+  options: JSX.Element[];
+}
+
+export const Dropdown: React.FC<DropdownProps> = ({
+  containerClassName,
```

Comment:
Should there even be outside class name and if so is `className` more in line with React convention?

Reply:
there is no need for `className` right now